modules/claims_api/app/workers/claims_api/vmbs_updater.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+require 'bgs'
+
+module ClaimsApi
+  class # frozen_string_literal: true
+
+    require 'sidekiq'
+    require 'bgs'
+    
+    module ClaimsApi
+      class VBMSUpdater
+        include Sidekiq::Worker
+    
+        def perform(power_of_attorney_id)
+          poa_form = ClaimsApi::PowerOfAttorney.find(power_of_attorney_id)
+          service = BGS::Services.new(
+            external_uid: poa_form.external_uid,
+            external_key: poa_form.external_key
+          )
+          service.vet_record.update_birls_record(
```

Comment:
i might have missed something but i thought we were needing to use the "update_poa_access" method of BGS?
https://vajira.max.gov/browse/API-6461?focusedCommentId=890592&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-890592

if this is also calling "update_birls_record" can we not just change the existing poa_updater job?
https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/claims_api/app/workers/claims_api/poa_updater.rb#L16

Reply:
more copy-paste nonsense. I even updated it with the right arguments but the wrong method 🤦‍♂️ correcting that too